azure-kusto-data/source/security.js

```diff
@@ -65,7 +66,7 @@ module.exports = class AadHelper {
                     if (err) {
                         return cb(err);
                     } else {
-                        console.log(tokenResponse.message);
+                        this.authCallback(tokenResponse);
```

Comment:
should fist check if a callback was set, otherwise let's call console.log instead?
another possibility is to do the same null check in the constructor:
this.authCallback = kcsb.AuthorizationCallback || (tokenResponse) => console.log(tokenResponse.message)

Reply:
You're right! Thanks :)